### ISSUE TITLE
oncoprint fails to load in rc branch

### ISSUE THREAD

**Issue by pieterlukasse** [cBioPortal/cbioportal#1339]
See 

http://www.cbioportal.org/beta/index.do?cancer_study_list=brca_tcga&cancer_study_id=brca_tcga&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=brca_tcga_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=brca_tcga_gistic&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=brca_tcga_cnaseq&case_ids=&patient_case_select=sample&gene_set_choice=prostate-cancer%3A-ar-signaling-%2810-genes%29&gene_list=SOX9+RAN+TNK2+EP300+PXN+NCOA2+AR+NRIP1+NCOR1+NCOR2&clinical_param_selection=null&tab_index=tab_visualize&Action=Submit

![image](https://cloud.githubusercontent.com/assets/2900303/15932429/f6df25ba-2e5b-11e6-8e00-cbf099382bf3.png)

**Comment by pieterlukasse** [comment]
@adamabeshouse @inodb : can we maybe get a screenshot test for oncoprint as well? Could be just a small one, e.g. with 4 samples, but that ensures the basic functionality is working.

**Comment by jjgao** [comment]
It is because of this one: https://github.com/cBioPortal/cbioportal/issues/1335

**Comment by pieterlukasse** [comment]
stack trace in response:

```
<html><head><title>Apache Tomcat/7.0.42 - Error report</title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}--></style> </head><body><h1>HTTP Status 500 - Request processing failed; nested exception is org.springframework.jdbc.BadSqlGrammarException: </h1><HR size="1" noshade="noshade"><p><b>type</b> Exception report</p><p><b>message</b> <u>Request processing failed; nested exception is org.springframework.jdbc.BadSqlGrammarException: </u></p><p><b>description</b> <u>The server encountered an internal error that prevented it from fulfilling this request.</u></p><p><b>exception</b> <pre>org.springframework.web.util.NestedServletException: Request processing failed; nested exception is org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
### The error may exist in org/cbioportal/persistence/mybatis/MutationMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select          &quot;mutation&quot;.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,         &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutationStatus&quot;,         &quot;mutation&quot;.&quot;TUMOR_REF_COUNT&quot; as &quot;referenceReadCountTumor&quot;,         &quot;mutation&quot;.&quot;NORMAL_REF_COUNT&quot; as &quot;referenceReadCountNormal&quot;,         &quot;mutation&quot;.&quot;TUMOR_ALT_COUNT&quot; as &quot;variantReadCountTumor&quot;,         &quot;mutation&quot;.&quot;NORMAL_ALT_COUNT&quot; as &quot;variantReadCountNormal&quot;,         &quot;mutation&quot;.&quot;VALIDATION_STATUS&quot; as &quot;validationStatus&quot;,          &quot;mutation_event&quot;.&quot;MUTATION_TYPE&quot; as &quot;mutationEvent.mutationType&quot;,         &quot;mutation_event&quot;.&quot;PROTEIN_CHANGE&quot; as &quot;mutationEvent.aminoAcidChange&quot;,         &quot;mutation_event&quot;.&quot;FUNCTIONAL_IMPACT_SCORE&quot; as &quot;mutationEvent.functionalImpactScore&quot;,         &quot;mutation_event&quot;.&quot;LINK_XVAR&quot; as &quot;mutationEvent.xvarLink&quot;,         &quot;mutation_event&quot;.&quot;LINK_PDB&quot; as &quot;mutationEvent.xvarLinkPdb&quot;,         &quot;mutation_event&quot;.&quot;LINK_MSA&quot; as &quot;mutationEvent.xvarLinkMsa&quot;,         &quot;mutation_event&quot;.&quot;CHR&quot; as &quot;mutationEvent.chr&quot;,         &quot;mutation_event&quot;.&quot;START_POSITION&quot; as &quot;mutationEvent.startPosition&quot;,         &quot;mutation_event&quot;.&quot;END_POSITION&quot; as &quot;mutationEvent.endPosition&quot;,         &quot;mutation_event&quot;.&quot;ONCOTATOR_PROTEIN_POS_START&quot; as &quot;mutationEvent.proteinStartPosition&quot;,         &quot;mutation_event&quot;.&quot;ONCOTATOR_PROTEIN_POS_END&quot; as &quot;mutationEvent.proteinEndPosition&quot;,         &quot;mutation_event&quot;.&quot;REFERENCE_ALLELE&quot; as &quot;mutationEvent.referenceAllele&quot;,         &quot;mutation_event&quot;.&quot;TUMOR_SEQ_ALLELE&quot; as &quot;mutationEvent.variantAllele&quot;,          &quot;gene&quot;.&quot;HUGO_GENE_SYMBOL&quot; as &quot;gene.hugoGeneSymbol&quot;,         &quot;gene&quot;.&quot;ENTREZ_GENE_ID&quot; as &quot;gene.entrezGeneId&quot;,          &quot;sample&quot;.&quot;STABLE_ID&quot; as &quot;sampleId&quot;,          &quot;cancer_study&quot;.&quot;CANCER_STUDY_IDENTIFIER&quot; as &quot;geneticProfile.studyId&quot;,          &quot;genetic_profile&quot;.&quot;STABLE_ID&quot; as &quot;geneticProfileId&quot;          from &quot;mutation&quot;         inner join &quot;mutation_event&quot; on &quot;mutation&quot;.&quot;MUTATION_EVENT_ID&quot; = &quot;mutation_event&quot;.&quot;MUTATION_EVENT_ID&quot;         inner join &quot;sample&quot; on &quot;mutation&quot;.&quot;SAMPLE_ID&quot; = &quot;sample&quot;.&quot;INTERNAL_ID&quot;         inner join &quot;gene&quot; on &quot;mutation&quot;.&quot;ENTREZ_GENE_ID&quot; = &quot;gene&quot;.&quot;ENTREZ_GENE_ID&quot;         inner join &quot;genetic_profile&quot; on &quot;genetic_profile&quot;.&quot;GENETIC_PROFILE_ID&quot; = &quot;mutation&quot;.&quot;GENETIC_PROFILE_ID&quot;         inner join &quot;cancer_study&quot; on &quot;genetic_profile&quot;.&quot;CANCER_STUDY_ID&quot; = &quot;cancer_study&quot;.&quot;CANCER_STUDY_ID&quot;                       inner join &quot;sample_list_list&quot; on &quot;sample_list_list&quot;.&quot;SAMPLE_ID&quot;=&quot;sample&quot;.&quot;INTERNAL_ID&quot;             inner join &quot;sample_list&quot; on &quot;sample_list&quot;.&quot;LIST_ID&quot;=&quot;sample_list_list&quot;.&quot;LIST_ID&quot;                    WHERE &quot;genetic_profile&quot;.&quot;STABLE_ID&quot; in                 (                     ?                 )                                           and &quot;gene&quot;.&quot;HUGO_GENE_SYMBOL&quot; in                 (                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 )                                                        and &quot;sample_list&quot;.&quot;STABLE_ID&quot; = ?
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
	org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:973)
	org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:863)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:647)
	org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:837)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
	org.mskcc.cbio.portal.util.XssFilter.doFilter(XssFilter.java:65)
	org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:186)
	org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
	org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:344)
	org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:261)
	org.apache.catalina.filters.CorsFilter.handleSimpleCORS(CorsFilter.java:302)
	org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:170)
</pre></p><p><b>root cause</b> <pre>org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
### The error may exist in org/cbioportal/persistence/mybatis/MutationMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: select          &quot;mutation&quot;.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,         &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutationStatus&quot;,         &quot;mutation&quot;.&quot;TUMOR_REF_COUNT&quot; as &quot;referenceReadCountTumor&quot;,         &quot;mutation&quot;.&quot;NORMAL_REF_COUNT&quot; as &quot;referenceReadCountNormal&quot;,         &quot;mutation&quot;.&quot;TUMOR_ALT_COUNT&quot; as &quot;variantReadCountTumor&quot;,         &quot;mutation&quot;.&quot;NORMAL_ALT_COUNT&quot; as &quot;variantReadCountNormal&quot;,         &quot;mutation&quot;.&quot;VALIDATION_STATUS&quot; as &quot;validationStatus&quot;,          &quot;mutation_event&quot;.&quot;MUTATION_TYPE&quot; as &quot;mutationEvent.mutationType&quot;,         &quot;mutation_event&quot;.&quot;PROTEIN_CHANGE&quot; as &quot;mutationEvent.aminoAcidChange&quot;,         &quot;mutation_event&quot;.&quot;FUNCTIONAL_IMPACT_SCORE&quot; as &quot;mutationEvent.functionalImpactScore&quot;,         &quot;mutation_event&quot;.&quot;LINK_XVAR&quot; as &quot;mutationEvent.xvarLink&quot;,         &quot;mutation_event&quot;.&quot;LINK_PDB&quot; as &quot;mutationEvent.xvarLinkPdb&quot;,         &quot;mutation_event&quot;.&quot;LINK_MSA&quot; as &quot;mutationEvent.xvarLinkMsa&quot;,         &quot;mutation_event&quot;.&quot;CHR&quot; as &quot;mutationEvent.chr&quot;,         &quot;mutation_event&quot;.&quot;START_POSITION&quot; as &quot;mutationEvent.startPosition&quot;,         &quot;mutation_event&quot;.&quot;END_POSITION&quot; as &quot;mutationEvent.endPosition&quot;,         &quot;mutation_event&quot;.&quot;ONCOTATOR_PROTEIN_POS_START&quot; as &quot;mutationEvent.proteinStartPosition&quot;,         &quot;mutation_event&quot;.&quot;ONCOTATOR_PROTEIN_POS_END&quot; as &quot;mutationEvent.proteinEndPosition&quot;,         &quot;mutation_event&quot;.&quot;REFERENCE_ALLELE&quot; as &quot;mutationEvent.referenceAllele&quot;,         &quot;mutation_event&quot;.&quot;TUMOR_SEQ_ALLELE&quot; as &quot;mutationEvent.variantAllele&quot;,          &quot;gene&quot;.&quot;HUGO_GENE_SYMBOL&quot; as &quot;gene.hugoGeneSymbol&quot;,         &quot;gene&quot;.&quot;ENTREZ_GENE_ID&quot; as &quot;gene.entrezGeneId&quot;,          &quot;sample&quot;.&quot;STABLE_ID&quot; as &quot;sampleId&quot;,          &quot;cancer_study&quot;.&quot;CANCER_STUDY_IDENTIFIER&quot; as &quot;geneticProfile.studyId&quot;,          &quot;genetic_profile&quot;.&quot;STABLE_ID&quot; as &quot;geneticProfileId&quot;          from &quot;mutation&quot;         inner join &quot;mutation_event&quot; on &quot;mutation&quot;.&quot;MUTATION_EVENT_ID&quot; = &quot;mutation_event&quot;.&quot;MUTATION_EVENT_ID&quot;         inner join &quot;sample&quot; on &quot;mutation&quot;.&quot;SAMPLE_ID&quot; = &quot;sample&quot;.&quot;INTERNAL_ID&quot;         inner join &quot;gene&quot; on &quot;mutation&quot;.&quot;ENTREZ_GENE_ID&quot; = &quot;gene&quot;.&quot;ENTREZ_GENE_ID&quot;         inner join &quot;genetic_profile&quot; on &quot;genetic_profile&quot;.&quot;GENETIC_PROFILE_ID&quot; = &quot;mutation&quot;.&quot;GENETIC_PROFILE_ID&quot;         inner join &quot;cancer_study&quot; on &quot;genetic_profile&quot;.&quot;CANCER_STUDY_ID&quot; = &quot;cancer_study&quot;.&quot;CANCER_STUDY_ID&quot;                       inner join &quot;sample_list_list&quot; on &quot;sample_list_list&quot;.&quot;SAMPLE_ID&quot;=&quot;sample&quot;.&quot;INTERNAL_ID&quot;             inner join &quot;sample_list&quot; on &quot;sample_list&quot;.&quot;LIST_ID&quot;=&quot;sample_list_list&quot;.&quot;LIST_ID&quot;                    WHERE &quot;genetic_profile&quot;.&quot;STABLE_ID&quot; in                 (                     ?                 )                                           and &quot;gene&quot;.&quot;HUGO_GENE_SYMBOL&quot; in                 (                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 ,                     ?                 )                                                        and &quot;sample_list&quot;.&quot;STABLE_ID&quot; = ?
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
; bad SQL grammar []; nested exception is com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
	org.springframework.jdbc.support.SQLErrorCodeSQLExceptionTranslator.doTranslate(SQLErrorCodeSQLExceptionTranslator.java:231)
	org.springframework.jdbc.support.AbstractFallbackSQLExceptionTranslator.translate(AbstractFallbackSQLExceptionTranslator.java:73)
	org.mybatis.spring.MyBatisExceptionTranslator.translateExceptionIfPossible(MyBatisExceptionTranslator.java:73)
	org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:371)
	com.sun.proxy.$Proxy273.selectList(Unknown Source)
	org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:198)
	org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:119)
	org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	com.sun.proxy.$Proxy280.getMutations(Unknown Source)
	org.cbioportal.persistence.mybatis.MutationMyBatisRepository.getMutations(MutationMyBatisRepository.java:19)
	org.cbioportal.service.impl.MutationServiceImpl.getMutations(MutationServiceImpl.java:20)
	org.mskcc.cbio.portal.service.ApiService.getGeneticProfileData(ApiService.java:363)
	org.mskcc.cbio.portal.service.ApiService$$FastClassBySpringCGLIB$$e4cf2f40.invoke(&lt;generated&gt;)
	org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:708)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:98)
	org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:262)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:95)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:644)
	org.mskcc.cbio.portal.service.ApiService$$EnhancerBySpringCGLIB$$37e7b5ba.getGeneticProfileData(&lt;generated&gt;)
	org.mskcc.cbio.portal.web.api.ApiController.getGeneticProfileData(ApiController.java:314)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:606)
	org.springframework.web.method.support.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:215)
	org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:132)
	org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:104)
	org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandleMethod(RequestMappingHandlerAdapter.java:749)
	org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:689)
	org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:83)
	org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:938)
	org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:870)
	org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:961)
	org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:863)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:647)
	org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:837)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
	org.mskcc.cbio.portal.util.XssFilter.doFilter(XssFilter.java:65)
	org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:186)
	org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
	org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:344)
	org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:261)
	org.apache.catalina.filters.CorsFilter.handleSimpleCORS(CorsFilter.java:302)
	org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:170)
</pre></p><p><b>root cause</b> <pre>com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.&quot;CENTER&quot; as &quot;sequencingCenter&quot;,
        &quot;mutation&quot;.&quot;MUTATION_STATUS&quot; as &quot;mutati' at line 3
	sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	com.mysql.jdbc.Util.handleNewInstance(Util.java:411)
	com.mysql.jdbc.Util.getInstance(Util.java:386)
	com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1052)
	com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3597)
	com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3529)
	com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1990)
	com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2151)
	com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2625)
	com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:2119)
	com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1362)
	org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172)
	org.apache.tomcat.dbcp.dbcp.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:172)
	org.apache.ibatis.executor.statement.PreparedStatementHandler.query(PreparedStatementHandler.java:59)
	org.apache.ibatis.executor.statement.RoutingStatementHandler.query(RoutingStatementHandler.java:73)
	org.apache.ibatis.executor.SimpleExecutor.doQuery(SimpleExecutor.java:60)
	org.apache.ibatis.executor.BaseExecutor.queryFromDatabase(BaseExecutor.java:267)
	org.apache.ibatis.executor.BaseExecutor.query(BaseExecutor.java:137)
	org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:90)
	org.apache.ibatis.executor.CachingExecutor.query(CachingExecutor.java:77)
	org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:108)
	org.apache.ibatis.session.defaults.DefaultSqlSession.selectList(DefaultSqlSession.java:102)
	sun.reflect.GeneratedMethodAccessor270.invoke(Unknown Source)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:606)
	org.mybatis.spring.SqlSessionTemplate$SqlSessionInterceptor.invoke(SqlSessionTemplate.java:358)
	com.sun.proxy.$Proxy273.selectList(Unknown Source)
	org.mybatis.spring.SqlSessionTemplate.selectList(SqlSessionTemplate.java:198)
	org.apache.ibatis.binding.MapperMethod.executeForMany(MapperMethod.java:119)
	org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:63)
	org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:52)
	com.sun.proxy.$Proxy280.getMutations(Unknown Source)
	org.cbioportal.persistence.mybatis.MutationMyBatisRepository.getMutations(MutationMyBatisRepository.java:19)
	org.cbioportal.service.impl.MutationServiceImpl.getMutations(MutationServiceImpl.java:20)
	org.mskcc.cbio.portal.service.ApiService.getGeneticProfileData(ApiService.java:363)
	org.mskcc.cbio.portal.service.ApiService$$FastClassBySpringCGLIB$$e4cf2f40.invoke(&lt;generated&gt;)
	org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:708)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:98)
	org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:262)
	org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:95)
	org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:644)
	org.mskcc.cbio.portal.service.ApiService$$EnhancerBySpringCGLIB$$37e7b5ba.getGeneticProfileData(&lt;generated&gt;)
	org.mskcc.cbio.portal.web.api.ApiController.getGeneticProfileData(ApiController.java:314)
	sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.lang.reflect.Method.invoke(Method.java:606)
	org.springframework.web.method.support.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:215)
	org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:132)
	org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:104)
	org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandleMethod(RequestMappingHandlerAdapter.java:749)
	org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:689)
	org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:83)
	org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:938)
	org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:870)
	org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:961)
	org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:863)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:647)
	org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:837)
	javax.servlet.http.HttpServlet.service(HttpServlet.java:728)
	org.mskcc.cbio.portal.util.XssFilter.doFilter(XssFilter.java:65)
	org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:186)
	org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
	org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:344)
	org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:261)
	org.apache.catalina.filters.CorsFilter.handleSimpleCORS(CorsFilter.java:302)
	org.apache.catalina.filters.CorsFilter.doFilter(CorsFilter.java:170)
</pre></p><p><b>note</b> <u>The full stack trace of the root cause is available in the Apache Tomcat/7.0.42 logs.</u></p><HR size="1" noshade="noshade"><h3>Apache Tomcat/7.0.42</h3></body></html>
```

**Comment by pieterlukasse** [comment]
seems to be a duplicate
